### ISSUE TITLE
`@defichain/testing` QOL utilities and setups

### ISSUE THREAD

**Issue by fuxingloh** [DeFiCh/jellyfish#431]
On the point of not using existing `client.xx.xx` where they are already `typed`, we don't need to be so strict about it. The best-case scenario would total isolation of concerns in each test case (file in this case) but that would just be overkill. If there is an existing typed method from the client. You can just use it. But for some obvious scenarios where we need isolation, e.g. it's not yet typed but we need assert it.

TLDR: it's okay to use `client.account.getAccount` if it's already merged instead of `container.call('getaccount', [])`.

> We need to look into this for greater testing QOL, but right now we don't have resources for it.

/area testing
/kind refactor

_Originally posted by @fuxingloh in https://github.com/DeFiCh/jellyfish/issues/399#issuecomment-870596525_
